tests/csapi/upload_keys_test.go

```diff
@@ -54,7 +54,7 @@ func TestUploadKey(t *testing.T) {
 
 		// sytest: Rejects invalid device keys
 		t.Run("Rejects invalid device keys", func(t *testing.T) {
-			runtime.SkipIf(t, runtime.Dendrite, runtime.Synapse) // Dendrite doesn't pass, Synapse has it blacklisted
+			runtime.SkipIf(t, runtime.Dendrite, runtime.Synapse) // Blacklisted on Synapse, Dendrite FIXME: https://github.com/matrix-org/dendrite/issues/2804
```

Comment:
Is there a corresponding Synapse issue?

Reply:
There is one here; https://github.com/matrix-org/complement/issues/517

I looked for one on synapse, but didn't open one there as I wasn't entirely sure what the reasoning was why these tests were blacklisted in the first place, so I opened one here to start that inquiry.